contributing/ci-omero.txt

```diff
@@ -171,19 +185,23 @@ under the :jenkinsview:`DEV` view tab of Jenkins.
         This job deploys the merge 5.3.x server (see
         :ref:`deployment_servers`)
 
+    :jenkinsjob:`WEB-DEV-merge-deploy`
+
+        This job deploys the merge 5.3.x web (see
+        :ref:`deployment_servers`)
+
     :jenkinsjob:`OMERO-DEV-merge-integration`
 
         This job runs the integration tests of OMERO
 
         #. Checks out :omero_scc_branch:`develop/merge/daily` 
         #. Builds OMERO.server and starts it
-        #. Runs the OMERO.java, OMERO.py and OMERO.web integration tests
+        #. Runs the OMERO.java and OMERO.py integration tests
         #. Archives the results
         #. Triggers downstream collection jobs:
            :term:`OMERO-DEV-merge-integration-broken`,
            :term:`OMERO-DEV-merge-integration-java`,
-           :term:`OMERO-DEV-merge-integration-python`,
-           :term:`OMERO-DEV-merge-integration-web`
+           :term:`OMERO-DEV-merge-integration-python`
```

Comment:
I think it make sense to do it when https://github.com/openmicroscopy/openmicroscopy/pull/4783 is merged in